pandas/tests/frame/test_quantile.py

```diff
@@ -97,19 +96,19 @@ def test_quantile_axis_parameter(self):
         with pytest.raises(ValueError, match=msg):
             df.quantile(0.1, axis="column")
 
-    def test_quantile_interpolation(self):
+    def test_quantile_interpolation(self, datetime_frame, int_frame):
```

Comment:
rather that having 2 fixtures that are tested independently, would it be straightforward to split this test?